File: Java/program/PetShop.java
// deklarasi kelas petshop
public class PetShop {
    // atribut private
    private String id, nama_produk, kategori_produk, harga_produk;

    /* konstruktor */

    // konstruktor tanpa parameter
    public PetShop() {
        // set isi atribut secara default
        this.id = "";
        this.nama_produk = "";
        this.kategori_produk = "";
        this.harga_produk = "";
    }

    // konstruktor dengan parameter
    public PetShop(String id, String nama_produk, String kategori_produk, String harga_produk) {
        // set isi atribut sesuai isi parameter
        this.id = id;
        this.nama_produk = nama_produk;
        this.kategori_produk = kategori_produk;
        this.harga_produk = harga_produk;
    }

    /* Getter dan Setter */

    // Get id
    public String getId() {
        return this.id;
    }

    // set id
    public void setId(String id) {
        this.id = id;
    }

    // Get nama_produk
    public String getNama_produk() {
        return this.nama_produk;
    }

    // set nama_produk
    public void setNama_produk(String nama_produk) {
        this.nama_produk = nama_produk;
    }

    // get kategori produk
    public String getKategori_produk() {
        return this.kategori_produk;
    }

    // set kategori produk
    public void setKategori_produk(String kategori_produk) {
        this.kategori_produk = kategori_produk;
    }

    // get harga produk
    public String getHarga_produk() {
        return this.harga_produk;
    }

    // set harga produk
    public void setHarga_produk(String harga_produk) {
        this.harga_produk = harga_produk;
    }
}

File: CPP/program/PetShop.cpp
// import library
#include <iostream>
#include <string>

// menggunakan standard namespace
using namespace std;

// deklarasi kelas PetShop
class Petshop
{
    // atribut private
private:
    string id, nama_produk, kategori_produk, harga_produk;

public:
    /* konstruktor */

    // konstruktor tanpa parameter
    Petshop()
    {
        // set isi atribut secara default
        this->id = "";
        this->nama_produk = "";
        this->kategori_produk = "";
        this->harga_produk = "";
    }

    // konstruktor dengan parameter
    Petshop(string id, string nama_produk, string kategori_produk, string harga_produk)
    {
        // set isi atribut sesuai isi parameter
        this->id = id;
        this->nama_produk = nama_produk;
        this->kategori_produk = kategori_produk;
        this->harga_produk = harga_produk;
    }

    /* Getter dan Setter*/

    // Get id
    string get_id()
    {
        return this->id;
    }

    // Set id
    void set_id(string id)
    {
        this->id = id;
    }
    // Get nama produk
    string get_nama_produk()
    {
        return this->nama_produk;
    }

    // Set nama produk
    void set_nama_produk(string nama_produk)
    {
        this->nama_produk = nama_produk;
    }

    // Get kategori produk
    string get_kategori_produk()
    {
        return this->kategori_produk;
    }

    // Set kategori produk
    void set_kategori_produk(string kategori_produk)
    {
        this->kategori_produk = kategori_produk;
    }

    // Get harga produk
    string get_harga_produk()
    {
        return this->harga_produk;
    }

    // Set harga produk
    void set_harga_produk(string harga_produk)
    {
        this->harga_produk = harga_produk;
    }

    /* desktruktor */

    // default
    ~Petshop() // desktruktor
    {
    }
};
